static/templates/message_edit_form.handlebars

```diff
@@ -5,7 +5,7 @@
     <div class="control-group no-margin">
         <label class="control-label edit-control-label" for="message_edit_topic">{{t "Topic" }}</label>
         <div class="controls edit-controls">
-          <input type="text" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
+          <input type="text" placeholder="{{t empty_topic }}" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
```

Comment:
@rishig  The doubt I was having was how to add translation tags to empty_topic_placeholder in compose.js.
Is there some resource I could see to understand better? Thank you :)